lib/bolt/project.rb

```diff
@@ -58,9 +74,26 @@ def initialize(path, type = 'option')
       @resource_types = @path + '.resource_types'
       @type = type
 
-      @project_file = @path + 'bolt-project.yaml'
-      @data = Bolt::Util.read_optional_yaml_hash(File.expand_path(@project_file), 'project') || {}
-      validate if load_as_module?
+      tc = Bolt::Config::TRANSPORT_CONFIG.keys & raw_data.keys
```

Comment:
I can't remember, but did we ever discuss the `transport` option? It seems a little weird that it's the only option that can be set in both `bolt-project.yaml` and `inventory.yaml`.

Reply:
From standup it looks like this was an oversight when we discussed this ticket. `transport` should only be supported in `inventory.yaml`.

Reply:
Hm, that's true - I think it makes sense to only have that be settable in inventory.